commands/build/lib/config.js

```diff
@@ -113,6 +114,32 @@ const config = ({
     external.push('@nebula.js/stardust');
   }
 
+  const output = () => {
+    if (argv.codeSplit) {
+      return {
+        banner,
+        format,
+        inlineDynamicImports,
+        dir: path.resolve(dir, outputFile.split('/')[0]),
+        name: outputName,
+        sourcemap,
+        globals: {
+          '@nebula.js/stardust': 'stardust',
+        },
+      };
+    }
+    return {
+      banner,
+      format,
+      file: path.resolve(dir, outputFile),
+      name: outputName,
+      sourcemap,
+      globals: {
+        '@nebula.js/stardust': 'stardust',
+      },
```

Comment:
if the option codeSplit is not sent in, output config remains the same

Reply:
Could make this a little cleaner (less code duplication) by having one output object with the default output props, and then, if `argv.codeSplit` you set `inlineDynamicImports` and `dir` accordingly before returning the output object from the function.